{"uid":"be09aaf6183d1909","name":"test_delete_favourites['Сало в соевом соусе вареное']","fullName":"tests.test_favorites_page.TestFavourites#test_delete_favourites","historyId":"5e43ca99d5f29c56de46e34a3b06b6a1","time":{"start":1689186461070,"stop":1689186461070,"duration":0},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55d452aa34e3 <unknown>\n#1 0x55d4527d2c76 <unknown>\n#2 0x55d4527fbd78 <unknown>\n#3 0x55d4527f8029 <unknown>\n#4 0x55d452836ccc <unknown>\n#5 0x55d45283647f <unknown>\n#6 0x55d45282dde3 <unknown>\n#7 0x55d4528032dd <unknown>\n#8 0x55d45280434e <unknown>\n#9 0x55d452a633e4 <unknown>\n#10 0x55d452a673d7 <unknown>\n#11 0x55d452a71b20 <unknown>\n#12 0x55d452a68023 <unknown>\n#13 0x55d452a361aa <unknown>\n#14 0x55d452a8c6b8 <unknown>\n#15 0x55d452a8c847 <unknown>\n#16 0x55d452a9c243 <unknown>\n#17 0x7f072d98fb43 <unknown>","statusTrace":"E   selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n      (unknown error: DevToolsActivePort file doesn't exist)\n      (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\n    Stacktrace:\n    #0 0x55d452aa34e3 <unknown>\n    #1 0x55d4527d2c76 <unknown>\n    #2 0x55d4527fbd78 <unknown>\n    #3 0x55d4527f8029 <unknown>\n    #4 0x55d452836ccc <unknown>\n    #5 0x55d45283647f <unknown>\n    #6 0x55d45282dde3 <unknown>\n    #7 0x55d4528032dd <unknown>\n    #8 0x55d45280434e <unknown>\n    #9 0x55d452a633e4 <unknown>\n    #10 0x55d452a673d7 <unknown>\n    #11 0x55d452a71b20 <unknown>\n    #12 0x55d452a68023 <unknown>\n    #13 0x55d452a361aa <unknown>\n    #14 0x55d452a8c6b8 <unknown>\n    #15 0x55d452a8c847 <unknown>\n    #16 0x55d452a9c243 <unknown>\n    #17 0x7f072d98fb43 <unknown>","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"browser","time":{"start":1689186461071,"stop":1689186461727,"duration":656},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55d452aa34e3 <unknown>\n#1 0x55d4527d2c76 <unknown>\n#2 0x55d4527fbd78 <unknown>\n#3 0x55d4527f8029 <unknown>\n#4 0x55d452836ccc <unknown>\n#5 0x55d45283647f <unknown>\n#6 0x55d45282dde3 <unknown>\n#7 0x55d4528032dd <unknown>\n#8 0x55d45280434e <unknown>\n#9 0x55d452a633e4 <unknown>\n#10 0x55d452a673d7 <unknown>\n#11 0x55d452a71b20 <unknown>\n#12 0x55d452a68023 <unknown>\n#13 0x55d452a361aa <unknown>\n#14 0x55d452a8c6b8 <unknown>\n#15 0x55d452a8c847 <unknown>\n#16 0x55d452a9c243 <unknown>\n#17 0x7f072d98fb43 <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1130, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/runner/work/testing_foodgram_project_react/testing_foodgram_project_react/conftest.py\", line 26, in browser\n    browser = webdriver.Chrome(\n              ^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/selenium/webdriver/chrome/webdriver.py\", line 84, in __init__\n    super().__init__(\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/selenium/webdriver/chromium/webdriver.py\", line 104, in __init__\n    super().__init__(\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 286, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 378, in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 440, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true}],"afterStages":[],"labels":[{"name":"story","value":"Тест удаления рецептов из избранного"},{"name":"feature","value":"Тесты добавления рецептов в избранное."},{"name":"tag","value":"favourites_test(scope='class')"},{"name":"tag","value":"run(order=5)"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_favorites_page"},{"name":"subSuite","value":"TestFavourites"},{"name":"host","value":"fv-az256-210"},{"name":"thread","value":"2518-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_favorites_page"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"data","value":"'Сало в соевом соусе вареное'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["run(order=5)","favourites_test(scope='class')"]},"source":"be09aaf6183d1909.json","parameterValues":["'Сало в соевом соусе вареное'"]}